[Error] Weapon Base Error

Just forwarding a report by a player

```

[ERROR] gamemodes/darkrp/entities/weapons/weapon_cs_base2/shared.lua:115: attempt to call method 'ResetBones' (a boolean value)
  1. unknown - gamemodes/darkrp/entities/weapons/weapon_cs_base2/shared.lua:115
```

Thanks.

--- a/entities/weapons/weapon_cs_base2/shared.lua
+++ b/entities/weapons/weapon_cs_base2/shared.lua
@@ -68,7 +68,7 @@ SWEP.MultiMode = false
 function SWEP:Initialize()
 	if CLIENT and IsValid(self.Owner) then
 		local vm = self.Owner:GetViewModel()
-		self:ResetBones(vm)
+		self:ResetDarkRPBones(vm)
 	end
 
 	self:SetWeaponHoldType("normal")
@@ -110,7 +110,7 @@ function SWEP:Holster()
 	if CLIENT then
 		if IsValid(self.Owner) then
 			local vm = self.Owner:GetViewModel()
-			self:ResetBones(vm)
+			self:ResetDarkRPBones(vm)
 		end
 		return
 	end
@@ -122,7 +122,7 @@ end
 function SWEP:Remove()
 	if CLIENT and IsValid(self.Owner) then
 		local vm = self.Owner:GetViewModel()
-		self:ResetBones(vm)
+		self:ResetDarkRPBones(vm)
 	end
 end
 
@@ -462,14 +462,14 @@ if CLIENT then
 		if not IsValid(self.Owner) then return end
 		local vm = self.Owner:GetViewModel()
 		
-		if self.ViewModelBoneManipulations then
-			self:UpdateBones(vm, self.ViewModelBoneManipulations)
+		if self.DarkRPViewModelBoneManipulations then
+			self:UpdateDarkRPBones(vm, self.DarkRPViewModelBoneManipulations)
 		else
-			self:ResetBones(vm)
+			self:ResetDarkRPBones(vm)
 		end
 	end
 
-	function SWEP:UpdateBones(vm, manipulations)
+	function SWEP:UpdateDarkRPBones(vm, manipulations)
 		if not IsValid(vm) or not vm:GetBoneCount() then return end
 
 		-- Fill in missing bone names. Things fuck up when we workaround the scale bug and bones are missing.
@@ -514,7 +514,7 @@ if CLIENT then
 		end 
 	end
 	
-	function SWEP:ResetBones(vm)	
+	function SWEP:ResetDarkRPBones(vm)	
 		if not IsValid(vm) or not vm:GetBoneCount() then return end
 		for i = 0, vm:GetBoneCount() - 1 do
 			vm:ManipulateBoneScale(i, Vector(1, 1, 1))

--- a/entities/weapons/weapon_m42/shared.lua
+++ b/entities/weapons/weapon_m42/shared.lua
@@ -25,7 +25,7 @@ SWEP.Category = "DarkRP (Weapon)"
 SWEP.ViewModel = "models/weapons/cstrike/c_rif_m4a1.mdl"
 SWEP.WorldModel = "models/weapons/w_rif_m4a1.mdl"
 SWEP.HoldType = "ar2"
-SWEP.ViewModelBoneManipulations = {
+SWEP.DarkRPViewModelBoneManipulations = {
 	["ValveBiped.Bip01_Spine4"] = { scale = Vector(1, 1, 1), pos = Vector(2, 0, 0), angle = Angle(0, 0, 0) },
 	["ValveBiped.Bip01_L_Hand"] = { scale = Vector(0.7, 0.7, 0.5), pos = Vector(-0.6, -0.6, 0), angle = Angle(17, -21, 0) },
 	["ValveBiped.Bip01_L_Finger0"] = { scale = Vector(1, 1, 1.5), pos = Vector(0, 0, 0), angle = Angle(0, -2, 0) },

